### DISCUSSION TITLE
Verifying GitHub Education while on Research Leave

### DISCUSSION THREAD

**Discussion by joanne-b-nortier** [community/community#146115] [GitHub Education]
### Github Education rejection due to Research Leave Location

Hi everyone! 
I'm a PhD based in the UK, but currently located elsewhere for an extended research trip. I tried to apply for the Github Education Pack by submitting both my university student card (with required expiration date) and my current affiliation's student card (with required expiration date at the end of the winter term next year) but I get rejected because of my gps location. I'm not sure how to get around this - any suggestions? 
I did do a search on the forum but couldn't find anything. 
Thanks in advance!

**Comment by sylviaD25** [comment]
I have the same problem. Have you solved yet?

**Reply by joanne-b-nortier** [reply]
Hi, as a workaround I was able to ask someone based locally to do the application from their device which worked fine.
